### ISSUE TITLE
Transfer coding for compression won't work

### ISSUE THREAD

**Issue by svgeesus** [w3c/IFT#115]
[Originally raised by Martin Thomson](https://lists.w3.org/Archives/Public/ietf-http-wg/2022AprJun/0279.html)

> The use of transfer coding for compression won't work in most cases.  Transfer coding isn't available in all versions of HTTP.

**Comment by garretrieger** [comment]
The spec [states](https://w3c.github.io/IFT/Overview.html#font-organization-compression) the the server must support either Content-Encoding or Transfer-Encoding. This should allow it to work with HTTP/1.1 through HTTP/3 servers. Realistically I would expect most implementations to use content-encoding.

**Comment by garretrieger** [comment]
Maybe a better way would be to always require "Content-Encoding" and have "Transfer-Encoding" be optional since a server that only supports "Transfer-Encoding" doesn't make much sense.

**Comment by svgeesus** [comment]
I agree that specifying `Content-Encoding` in the spec would be best. Must admit I was unaware that `Transfer Coding` had been dropped.

@martinthomson does that sound good to you?

**Comment by martinthomson** [comment]
Yeah, I would just drop any mention of transfer coding.
